book/glossary.md

```diff
@@ -83,59 +101,86 @@ __HTML__.
 __Element__: Most __Nodes__ in the __DOM__ tree are Elements (except for
 text and the document object). (Inherits from __Node__.)
 
-__Focus__: the property of an __element__ being the highlight, or "focus",
+__Event__: A way for JavaScript to observe that something has happened on the
+document, and customize its results.
+
+__Focus__: Ahe property of an __element__ being the highlight, or "focus",
 of user interaction, and therefore receiving keyboard events and being
 visually highlighted on the screen.
 
-__IFrame__: a way of embedding a child __document__ within a parent, through
+__Font__: A particular stylistic way of drawing a particular human language to
+computer screens. Arial is one common example for Latin-based langauges.
```

Comment:
```suggestion
computer screens. Times New Roman is one common example for Latin-based languages.
```

At least in the US, Times New Roman seems better known to my students. (Something something high school essays.) No idea what's better known internationally? Helvetica?